.github/workflows/examples-ci.yaml

```diff
@@ -28,65 +28,63 @@ jobs:
     name: Examples CI (${{ matrix.os }}, py-${{ matrix.python-version }}, rdkit=${{ matrix.include-rdkit }}, openeye=${{ matrix.include-openeye }}, dgl=${{ matrix.include-dgl }}), pydantic=${{ matrix.pydantic-version }}
     runs-on: ${{ matrix.os }}
     strategy:
-        fail-fast: false
-        matrix:
-          os: [ubuntu-latest]
-          python-version: ["3.11", "3.12"]
-          pydantic-version: ["2"]
-          include-rdkit: [true]
-          include-openeye: [false]
-          include-dgl: [true]
-
+      fail-fast: false
+      matrix:
+        os: [macOS-12, ubuntu-latest]
```

Comment:
`macos-12` is being removed next month 😞

Reply:
Is `macos-latest` appropriate?

Reply:
Yeah I think so, it's no longer intel but AmberTools doesn't work on macOS-13. Thanks so much for looking at this!